jams/eval.py

```diff
@@ -32,6 +32,9 @@ def validate_annotation(ann, namespace):
     '''Validate that the annotation has the correct namespace,
```

Comment:
Cool. You might want to update the docstring of the function, but otherwise everything looks good.